proto/sync/sync.proto

```diff
@@ -57,9 +57,9 @@ message Key {
   bytes value = 2;
 }
 
+// If the message is empty, the value is considered to be "nothing".
```

Comment:
I think the term normally used is [presence](https://protobuf.dev/programming-guides/field_presence/)
```suggestion
// If the message is present, the value is considered to be "something".
```